lib/twitter/twitter_api.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:convert';
+import 'package:cake_wallet/twitter/twitter_user.dart';
+import 'package:http/http.dart' as http;
+import 'package:cake_wallet/.secrets.g.dart' as secrets;
+
+class TwitterApi {
+  static const twitterBearerToken = secrets.twitterBearerToken;
+  static const httpsScheme = 'https';
+  static const apiHost = 'api.twitter.com';
+  static const userPath = '/2/users/by/username/';
```

Comment:
**Minor:** 
Those can be private to prevent any change to them from outside by mistake